core/src/main/java/hudson/Util.java

```diff
@@ -1073,11 +1073,11 @@ public static void touch(@NonNull File file) throws IOException {
     /**
      * Copies a single file by using Ant.
      *
-     * @deprecated since TODO; use {@link Files#copy(Path, Path, CopyOption...)} directly
+     * @deprecated since 2.335; use {@link Files#copy(Path, Path, CopyOption...)} directly
```

Comment:
Weird to have this here, hopefully Java 11 in core will be the saint to move to `@Deprecated(since = "version")` 🙏🏻